site/app/templates/submission/homework/SubmitBox.twig

```diff
@@ -1,6 +1,11 @@
 {% import _self as self %}
 
-<div class="content">
+{# Display the gradeable_message if one exists at the top of the page #}
+{% if has_gradeable_message %}
+    <div class='content' style='background-color: #006398; color: white;'><p>{{ gradeable_message|raw }}</p></div>
```

Comment:
Would highly recommend using a css class and putting the style elements in server.css